README.md

```diff
@@ -18,6 +18,8 @@ make presubmit
 ```
 and check its output to make sure your code coverage figures are in line with the set target and that there are no newly introduced lint problems.
 
+Note: You will need mockgen for testing: `go install github.com/golang/mock/mockgen@v1.5.0`, and the usual `go mod tidy` to fetch dependencies.
```

Comment:
`mockgen` is a standalone CLI: I don't think we need to update the module's dependencies after installing it.